### PR TITLE
Tool error checking via profile in Kraken Taxonomy Report

### PR DESCRIPTION
**Pull request by peterjc**
This is a relatively trivial update to the Kraken Taxonomy Report, but perhaps worth applying anyway given the tests pass.

(I actually just wanted to trigger a TravisCI build and test on another tool using its own `test-data/example.loc` file to confirm that was working properly while debugging something else).

### CODE REVIEW

**Comment by peterjc** [comment]
P.S. If the current recommendation is to use ``<command detect_errors="exit_code">`` rather than the profile feature, I'm happy to do that. I have a lot of my own tools where I can update this.

**Comment by blankenberg** [comment]
xref https://github.com/galaxyproject/tools-iuc/pull/945

**Comment by bgruening** [comment]
@blankenberg can we get those two PR resolved?

**Comment by bgruening** [comment]
@blankenberg

File: tools/kraken_taxonomy_report/kraken_taxonomy_report.xml [hunk 1]
```diff
@@ -1,13 +1,8 @@
-<?xml version="1.0"?>
-<tool id="kraken_taxonomy_report" name="Kraken taxonomic report" version="0.0.2">
+<tool id="kraken_taxonomy_report" name="Kraken taxonomic report" version="0.0.3" profile="16.04">
```
Comment (bgruening) [inline tools/kraken_taxonomy_report/kraken_taxonomy_report.xml:1]: raise the provile?

**Review by bgruening — APPROVED** [review]

**Pull request merged**